components/Shell.tsx

```diff
@@ -53,7 +53,7 @@ function useRedirectToLoginIfUnauthenticated() {
   useEffect(() => {
     if (!loading && !session) {
       router.replace({
-        pathname: "/auth/login",
+        pathname: "https://theskills.com/sign-in",
```

Comment:
To streamline dev and staging testing, can you change this to relative url `${process.env.THETIS_SITE_HOST}/sign-in}`

Reply:
Yeah good call